Services/TermsOfService/classes/class.ilTermsOfServiceTableDatabaseDataProvider.php

```diff
@@ -88,7 +88,7 @@ public function getList(array $params, array $filter) : array
                 throw new InvalidArgumentException('Please provide a valid numerical offset.');
             }
 
-            $this->db->setLimit($params['limit'], $params['offset']);
+            $this->db->setLimit($params['limit'], $params['offset']);  //PHP8-Review: Parameter #2 $offset of method ilDBInterface::setLimit() expects int, float|int|string given
```

Comment:
This is a `False Positive`. The values are retrieved by `$params['offset'] = $this->getOffset()` and `$params['limit'] = $this->getLimit()` (`ilTable2GUI`) and passed to this class. Both return values are `int`.